webpack.config.js

```diff
@@ -1,26 +1,12 @@
 const path = require("path");
-// MiniCssExtractPlugin is a plugin that extracts CSS into separate files. It creates a CSS file per JS file which contains CSS.
-const MiniCssExtractPlugin = require("mini-css-extract-plugin");
-// HtmlWebpackPlugin is a plugin that simplifies creation of HTML files to serve your webpack bundles.
-const HtmlWebpackPlugin = require("html-webpack-plugin");
+// PugPlugin is a plugin that simplifies creation of HTML files to serve your webpack bundles.
+const PugPlugin = require("pug-plugin");
 // CleanWebpackPlugin is a plugin that removes/cleans build folders and unused assets when rebuilding.
 const { CleanWebpackPlugin } = require("clean-webpack-plugin");
 
-// multipage website
-// Pages array for HtmlWebpackPlugin
-const pages = ["index", "about"];
-
 // Webpack configuration
 module.exports = {
-  // Define dynamically an entry for each page
-  // -------------------------------
-  entry: pages.reduce((config, page) => {
-    config[page] = `./src/views/${page}/${page}.js`;
```

Comment:
The entrypoint is a Pug template where are specified corresponding source JS and SCSS files.